### PR TITLE
Update docs for embedded components

### PR DESCRIPTION
**Pull request by teresam856**
Found during pairing with @rbuels 

After updating the website for desktop release,  the embedded components docs got buried and it can be hard to quickly find storybook links.

Screen Shot 2021-11-09 at 5 34 07 PM

Clicking on links above does not direct you to the appropriate section in the downloads docs. It can be confusing to click on embedded components and see desktop as the first thing. The link to "other platforms" directs you to the latest github tag/release? Maybe this could be a link to the introduction page of the docs.

Screen Shot 2021-11-09 at 4 10 55 PM

Could be worth making a small little page for embedded docs once we add [Circular View embedded docs](https://github.com/GMOD/jbrowse-components/issues/2462)

Related Issue [Make Embedded docs more visible](https://github.com/GMOD/jbrowse-components/issues/2507)

### CODE REVIEW

**Comment by teresam856** [comment]
Proposal to reorganize the side bar:
Before
intro_before


After (added section for embedded components)
intro_after

**Comment by cmdcolin** [comment]
why the sidebar change?

**Comment by teresam856** [comment]
> why the sidebar change?

Grouping docs that are similar makes it easier to find what you need.  Also easier for the eye to find.

**Comment by cmdcolin** [comment]
it looks like it moves the quickstart guides down though, making it less clear

i can see perhaps having all the nested categories together looks nice but is the organization is better?

**Comment by teresam856** [comment]
We could move the quick start guides back towards the top, since those are there for easy access.  I still think reducing and even grouping the available docs could make it easier for users to find the resources they need.  

I would even suggest removing 'JBrowse 2 combined PDF' since it is already mentioned in the 'Jbrowse2 combined guide'

File: website/src/pages/download.mdx [hunk 1]
```diff
@@ -139,6 +139,8 @@ export const DownloadCardContainer = () => {
 To get started with JBrowse Web, we recommend using the jbrowse CLI tools.
 You can also use the jbrowse CLI for things like adding tracks too.
 
+### JBrowse2 CLI tools
```
Comment (cmdcolin) [inline website/src/pages/download.mdx:142]: might just say "JBrowse CLI tools" or if using 2, "JBrowse 2" with a space

**Comment by cmdcolin** [comment]
I'd be ok if "JBrowse combined guide" was removed and even "JBrowse combined PDF guide" if the PDF was linked from the docs landing page e.g. introduction.md

**Comment by carolinebridge-oicr** [comment]
Agree with leaving quick start near the top, and FAQ at the end is in-line with typical documentation indexing, that is a good change

**Comment by teresam856** [comment]
Update from backlog grooming: 
* remove the JBrowse Combined PDF Guide
* add section for Embedded Components which will contain the LGV docs and storybook links and later on any circular view docs

**Comment by teresam856** [comment]
Update:
embedded_components_docs_page

File: website/docs/embedded_components.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+id: embedded_components
+title: Embedded components
+toplevel: true
+---
+
+JBrowse 2 is a pluggable open-source platform for visualizing, integrating, and sharing biological data.
+
+## JBrowse2 React Linear Genome View
```
Comment (cmdcolin) [inline website/docs/embedded_components.md:9]: As before, the "JBrowse2" could remove the 2

**Review by carolinebridge-oicr — APPROVED** [review]

**Comment by rbuels** [comment]
Can we get the embedded circular view on here as well?

**Pull request merged**